sync/src/main/java/org/bf2/sync/informer/InformerManager.java

```diff
@@ -46,6 +51,19 @@ protected void onStart() {
         managedAgentInformer = resourceInformerFactory.create(ManagedKafkaAgent.class, client.resources(ManagedKafkaAgent.class).inAnyNamespace(),
                 CustomResourceEventHandler.of(controlPlane::updateAgentStatus));
 
+        secretInformer = resourceInformerFactory.create(Secret.class, client.secrets().inAnyNamespace().withLabels(OperandUtils.getDefaultLabels()),
```

Comment:
I wonder if there should be an additional label on the master secret and this informer scoped to only those secrets.

Reply:
yes, that makes sense, while creating the secret can give additional label to filter on